backend-lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zcash-android-wallet-sdk"
-version = "2.2.9"
+version = "2.2.10"
```

Comment:
@nuttycom should we now keep this `cargo.toml` version field aligned with the main library version field in `gradle.properties`(see changes in this PR)? I noticed that you recently started to bump it, it was `0.0.4` for a longer time, though?